go/cmd/vtctldclient/command/throttler.go

```diff
@@ -17,32 +17,47 @@ limitations under the License.
 package command
 
 import (
+	"time"
+
 	"github.com/spf13/cobra"
 
 	"vitess.io/vitess/go/cmd/vtctldclient/cli"
 
+	"vitess.io/vitess/go/vt/logutil"
+	"vitess.io/vitess/go/vt/proto/topodata"
```

Comment:
Nit, but we would generally use `topodatapb` as the import name.

Reply:
fixed